packages/@coorpacademy-components/src/molecule/learner-skill-card/index.js

```diff
@@ -0,0 +1,170 @@
+import Provider from '../../atom/provider';
+import style from './style.css';
+
+const LearnerSkillCard = (props, context) => {
+  const {
+    'aria-label': ariaLabel,
+    skillCourses,
+    skillQuestions,
+    skillTitle,
+    skillAriaLabel,
+    completedCourses,
+    buttonReviewLabel,
+    buttonReviewAriaLabel,
+    buttonExploreLabel,
+    buttonExploreAriaLabel,
+    onReviewClick,
+    onExploreClick
+  } = props;
+  const {skin, translate} = context;
+  const [hovered, setHovered] = useState(false);
+  const primarySkinColor = getOr('#0061FF', 'common.primary', skin);
+
+  const courses = translate('courses');
+  const questions = translate('questions');
+  const skillFocus = translate('skill_focus');
+  const coursesCompleted = translate('courses_completed');
+
+  const handleMouseOver = useCallback(() => setHovered(true), [setHovered]);
+
+  const handleMouseLeave = useCallback(() => setHovered(false), [setHovered]);
+
+  const buttonReviewProps = {
+    customStyle: {
+      backgroundColor: '#FFF',
+      transition: 'background-color 0.15s ease-in-out, color 0.15s ease-in-out'
+    },
+    onReviewClick,
+    'aria-label': buttonReviewAriaLabel ? `${skillTitle}, ${buttonReviewAriaLabel}` : skillTitle,
+    label: buttonReviewLabel,
+    'data-name': 'learner-skill-card-review-button'
+  };
+
+  const buttonExploreProps = {
+    customStyle: {
+      backgroundColor: hovered ? primarySkinColor : convert(`color(${primarySkinColor} a(0.07))`),
+      color: hovered ? '#FFFFFF' : primarySkinColor,
+      transition: 'background-color 0.15s ease-in-out, color 0.15s ease-in-out'
+    },
+    onExploreClick,
+    'aria-label': buttonExploreAriaLabel ? `${skillTitle}, ${buttonExploreAriaLabel}` : skillTitle,
+    label: buttonExploreLabel,
+    'data-name': 'learner-skill-card-explore-button',
+    icon: {
+      position: 'left',
+      faIconName: 'compass',
+      faIconBackgroundColor: hovered
+        ? primarySkinColor
+        : convert(`color(${primarySkinColor} a(0.07))`),
+      faIconColor: hovered ? '#FFFFFF' : primarySkinColor,
+      faIconSize: 16
+    }
+  };
+
+  const completedPercentage = Number.parseInt((completedCourses / skillCourses) * 100);
+
+  const ProgressBar = useCallback(() => {
+    const progressBarColor = '#3EC483';
+    const inlineProgressValueStyle = {
+      backgroundColor: progressBarColor,
+      width: `${completedPercentage}%`
+    };
+
+    return (
+      <div className={style.progressWrapper}>
+        <div
+          data-name="progress"
+          className={style.progress}
+          style={inlineProgressValueStyle}
+          role="progressbar"
+          aria-label={get('progression', ariaLabel)}
+        />
+      </div>
+    );
+  }, [completedPercentage, ariaLabel]);
+
+  return (
+    <div
+      className={style.learnerSkillCardWrapper}
+      data-name="learner-skill-card-wrapper"
+      aria-label={ariaLabel}
+    >
+      <div className={style.skillCoursesAndQuestionsWrapper}>
```

Comment:
Note, component should support missing kPis (courses, questions, and so on) and hide the section if missing

Reply:
(💭  maybe, could be interesting to group all these kpis in a nested objects, like `counts`/`metrics`: `{courses, questions, completedCourses}`)

Reply:
thank you, I've updated